pkgs/applications/audio/jamulus/default.nix

```diff
@@ -0,0 +1,36 @@
+{ mkDerivation, stdenv, fetchFromGitHub, pkgconfig, qtscript, qmake, libjack2
+}:
+
+mkDerivation {
+  pname = "jamulus";
+  version = "3.5.3";
+  src = fetchFromGitHub {
+    owner = "corrados";
+    repo = "jamulus";
+    rev = "5385f4f113884537c18109e01791149754fe96d5";
```

Comment:
It is preferred to use a tag `rev = "v${version}"` when available. It will make updating easier in the future.

Reply:
changed to the tag name. Or is there a good way to get "3_5_3" from the version "3.5.3"? (or change the version string?